ST_initialization.c

```diff
@@ -0,0 +1,301 @@
+/***********************************************************************/
+// ST_initialization.c 
+//      Contains definitions of all functions related to initialization.
+//      The current initialization methods are _run_seed_initialization
```

Comment:
@kpalmqui I believe that setting up the logic and structure for _run_seed_initialization will make it easier for future programmers to work with he code base. Since the function already exists and the logic behind when to call the function is in place a future programer would only have to work inside the _run_seed_initialization function as opposed to also figuring out when they should call the function and where they should put the function. 

One thing that I should definitely do is update the documentation regarding seed dispersal in ST_initialization.c to make it clear what a future programmer has to do.

Of course this is completely up to you. If you would like I could remove the function and logic entirely.

Reply:
We have decided to retain the functionality/code for now and re-evaluate the decision in the next development branch.